### DISCUSSION TITLE
Provide certification for digital logic design

### DISCUSSION THREAD

**Discussion by tachyons** [CircuitVerse/CircuitVerse#3745] [Ideas]
We have circuitverse simulation platform and interactive learning environment, one natural evolution is to provide certification which can also be used a potential revenue source. We can also integrate with https://openbadges.org/.

@CircuitVerse/circuitverse-core-team

**Comment by ayan-biswas0412** [comment]
Great idea @tachyons but how it could be a revenue source ?

**Reply by tachyons** [reply]
We need to charge for certificates, course material will remain free to access

**Reply by 092vk** [reply]
@tachyons I have proposed a similar idea on GSOC 2025 discussion , But currently we don't have a great learning path , which user can follow and design different digital circuits .
 
- I have proposed a game based learning path like we have in candy crush , where on each level user has to design a circuit. 
- It will start with simple circuits like full adder , encoder , decoder etc and move on to implementing real life digital circuits like that of traffic light. 
- It will cover 50 circuits , the user will collect badges , titles and prizes on completing them .
-  Which they can use to flex on linkdin  and show them to their potential recruiters.
-  We can introduce premium access for small fee where user will get other features like exam prep(company-wise questions) , certifications, advanced circuits and other benefits.
- On premium access the teachers and companies will be able to host their own contests and virtual lab
